base/number.jl

```diff
@@ -112,10 +112,36 @@ copysign(x::Real, y::Real) = ifelse(signbit(x)!=signbit(y), -x, +x)
 conj(x::Real) = x
 transpose(x::Number) = x
 ctranspose(x::Number) = conj(x)
-inv(x::Number) = one(x)/x
 angle(z::Real) = atan2(zero(z), z)
 
 """
+    inv(x)
+
+Return the multiplicative inverse of `x`, such that `x*inv(x)` or `inv(x)*x`
+yields [`one(x)`](@ref) (the multiplicative identity).
+
+If `x` is a number, this is essentially the same as `one(x)/x`, but for
```

Comment:
It need not be identical under roundoff errors, nor is it necessarily computationally identical.  Hence "essentially"